### DISCUSSION TITLE
Running `npm run {command}` from within NvChad

### DISCUSSION THREAD

**Discussion by nstanard** [NvChad/NvChad#1398] [Ideas]
Hello, I'm currently looking for a way to run `npm` commands from within NvChad. I'm not seeing any open discussions or instructions in the docs so far. I'm creating this discussion thread to keep track of my thoughts for how to do this.

My ideal scenario would be...

While nvim is open I can `:Npm{Run|Install|Update}` and then create shortcut mappings to trigger specific `:NpmRun {alias}` commands within the projects I manage.

**Comment by siduck** [comment]
@nstanard 

make a keybind for this 

```lua
require("nvterm.terminal").send('npm install')
```

check https://github.com/NvChad/nvterm#additional-functionality for more info

**Reply by nstanard** [reply]
Thanks, I'll give this a try!

**Reply by nstanard** [reply]
https://github.com/nstanard/nvchad-custom/blob/main/custom/mappings.lua#L7

👏 👏 👏
